### PR TITLE
Evaluation results of v0.4.0

### PR DESCRIPTION
**Pull request by zimmski**

### CODE REVIEW

File: docs/reports/v0.4.0/categories.svg [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+L 510 75" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="81" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">60</text><path  d="M 505 73
+L 510 73" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="79" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">61</text><path  d="M 505 70
+L 510 70" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="76" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">62</text><path  d="M 505 68
+L 510 68" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="74" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">63</text><path  d="M 505 66
+L 510 66" style="stroke-width:0;stroke:rgba(51,51,51,1.0);fill:none"/><text x="515" y="72" style="stroke-width:0;stroke:none;fill:rgba(51,51,51,1.0);font-size:12.8px;font-family:'Roboto Medium',sans-serif">64</text><path  d="M 20 225
```
Comment (bauersimon) [inline docs/reports/v0.4.0/categories.svg:111]: Should we delete some y axis ticks?

**Review by bauersimon — APPROVED** [review]

File: docs/reports/v0.4.0/README.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+-   `openrouter/openai/gpt-4-32k-0314`
+-   `openrouter/openai/gpt-4`
+-   `openrouter/openai/gpt-4-0314`
+-   `openrouter/mistralai/mistral-7b-instruct:nitro`
+-   `openrouter/openrouter/auto`
+-   `openrouter/openai/gpt-4-vision-preview`
+-   `openrouter/anthropic/claude-3-sonnet:beta`
+-   `symflower/symbolic-execution`
+-   `openrouter/openai/gpt-3.5-turbo-1106`
+-   `openrouter/openai/gpt-3.5-turbo-0125`
```
[the remark this answers is not in the document]
Comment (zimmski) [reply docs/reports/v0.4.0/README.md:196]: why? Haven't looked into that one

File: docs/reports/v0.4.0/categories.svg (hunk 1 above)
Comment (zimmski) [reply docs/reports/v0.4.0/categories.svg:111]: You can do that with a follow up if you like.

Comment (zimmski) [reply docs/reports/v0.4.0/categories.svg:111]: No. just delete the y axis and give the bars numbers. that is much better to read

File: docs/reports/v0.4.0/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bauersimon) [reply docs/reports/v0.4.0/README.md:196]: having all the openai, google, microsoft and meta models in the upper field does at least not surprise me

**Pull request merged**